doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -134,6 +138,215 @@ Examine the FluentBit sidecar's STDOUT to see logs for Ray's component processes
 kubectl logs raycluster-complete-logs-head-xxxxx -c fluentbit
 ```
 
+(kuberay-fluentbit-daemonset)=
+## Setting up logging DaemonSet with Fluent Bit
+
+In this section, we give a walkthrough of how to set up log-emitting [Fluent Bit][FluentBit] DaemonSet for Ray pods.
```

Comment:
It would be helpful to include a brief description of what Fluent Bit is and what it's going to do for us, even though we link to fluentbit's website